tomviz/MainWindow.ui

```diff
@@ -160,7 +167,7 @@
         <bool>true</bool>
        </property>
        <property name="columnCount">
-        <number>2</number>
+        <number>1</number>
```

Comment:
This column thing again - can you fix it up so that it stops changing this down to 1? Failing that just remove this change for now.